README.md

```diff
@@ -1,74 +1,65 @@
 Event Hub
 =========
 Event Hub is an offshoot website of of Bright Events collaborative development training, hosted by Andela. The Site allows users to view and manage their own events as well as inform potential attendants of the event's existence. Visit our link at https://kalela.github.io/BrightEvents/.
```

Comment:
I would like you take recommendations raised in the issue to make your readme more informative.